suite.py

```diff
@@ -1256,7 +1258,7 @@ def build_test_cmake(self, test, opts="",  outfile=None):
             else:
                 # Copy and rename executable to test dir
                 shutil.move(f"{path_to_exe}",
-                            f"{self.source_dir}/{test.buildDir}/{test.name}.ex")
+                            f"{test.output_dir}/{test.name}.ex")
```

Comment:
Maybe it was `{self.source_dir}/{test.buildDir}` after all and only the broken `get_recent_filename` confused me :)